src/services/Api.ts

```diff
@@ -57,15 +57,13 @@ export class Api<T extends PrefectConfig = PrefectConfig> {
     }, {})
   }
 
-  protected removeLeadingAndTrailingSlashes(input: string): string {
-    return input.replace(/^(\/)|(\/)$/g, '')
-  }
-
   protected combinePath(route: string | undefined): string {
+    const repeatingSlashes = /(\/+)/g
```

Comment:
Nit: So this will get all slashes eh? Maybe a better name is `slashes` or `findSlashes`, idk.